ios/RNBranch.m

```diff
@@ -51,22 +50,14 @@ + (Branch *)branch
         static Branch *instance;
         static dispatch_once_t once = 0;
         dispatch_once(&once, ^{
-            RNBranchConfig *config = RNBranchConfig.instance;
-
-            // YES if either [RNBranch useTestInstance] was called or useTestInstance: true is present in branch.json.
-            BOOL usingTestInstance = useTestInstance || config.useTestInstance;
-            NSString *key = branchKey ?: config.branchKey ?: usingTestInstance ? config.testKey : config.liveKey;
-
-            if (key) {
+            if (branchKey) {
```

Comment:
Is there a scenario where `branchKey` has a non-null?
With the migration to native SDKs, branchKey will be set to whatever branch.json has or fallback to plist file.
If branchKey is set at this point, it will override json and plist. Is that ok?

Reply:
You're stumbling into some unreleased code. One thing that has been requested by RN users for a long time is to be able to set keys from JS, but that requires waiting for JS to load and deferring native initialization. Android has always posed limitations there. Unfortunately things like that have bitten us in the past when quietly released only for iOS. People assume they're supported on Android too, and then we have to make it so. So I've commented out some bits here and there, but it is basically possible to initialize the native iOS SDK from JS. That's what the `initializeBranch:` method is for around line 352.